src/Core/System/Snippet/DataTransfer/Metadata/MetadataCollection.php

```diff
@@ -0,0 +1,76 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\System\Snippet\DataTransfer\Metadata;
+
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Struct\Collection;
+
+/**
+ * @internal
+ *
+ * @extends Collection<MetadataEntry>
+ */
+#[Package('discovery')]
```

Comment:
I personally would have chosen `fundamentals@discovery`. Do you now how we seperate this?